.github/workflows/ci.yaml

```diff
@@ -259,7 +259,7 @@ jobs:
       - uses: ory/ci/releaser/render-version-schema@master
         with:
           token: ${{ secrets.ORY_BOT_PAT }}
-          schema-path: embedx/config.schema.json
+          schema-path: .schema/config.schema.json
```

Comment:
Can you please change this back and remove the file `.schema/config.schema.json`?